config.toml.example

```diff
@@ -173,9 +173,7 @@
 
 # Python interpreter to use for various tasks throughout the build, notably
 # rustdoc tests, the lldb python interpreter, and some dist bits and pieces.
-#
-# Defaults to the Python interpreter used to execute x.py.
-#python = "python"
+#python = "python" # defaults to the Python interpreter used to execute x.py
```

Comment:
```suggestion
#
# defaults to the Python interpreter used to execute x.py
#
#python = "python" 
```

I find it hard to read with the comment at the end of the line like this. It's hard to tell that this line is a setting and not just another sentence at the end of the previous paragraph.

The same applies below.

Reply:
Yeah, that's true, it did seem to blend in to me.